src/Plugin/Condition/DataListContains.php

```diff
@@ -52,8 +49,7 @@ public function evaluate() {
       }
       return FALSE;
     }
-
-    return in_array($item, $list);
+    return in_array($item, $list);    
```

Comment:
This looks like an area that varies a bit from plugin to plugin.  Similar plugins appear to put return statements flush with the "}" of the previous line.  This is not universal, but most plugins that have this kind of code structure omit the extra line.